### PR TITLE
Assignment 4 @Malcolm-Shepherd

### PR DESCRIPTION
**Pull request by Clockknight**

### CODE REVIEW

**Review by alowe4 — COMMENTED** [review]
Make all pages responsive ❌
        1080p desktop ✅
        iPad Air ✅
        Samsung Galaxy S20 Ultra Super Mega Excellent ❌
        IPhone SE ❌
    Create a improved landing page and site theme (Background, make it look attractive to play) ✅
        Theme should extend all pages (Maybe not the ad page 😁) ✅
    Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
    Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
    Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌[should list avg Time]
        It should indicate if the user has played that day ❌
        Clicking on the item should take the player to that game (Extra Credit) ❌

**Review by DaisyPalencia — COMMENTED** [review]
Features
Make all pages responsive ❌
1080p desktop ✅
iPad Air ✅
Samsung Galaxy S20 Ultra Super Mega Excellent ❌
IPhone SE ❌
Create a improved landing page and site theme (Background, make it look attractive to play) ✅
Theme should extend all pages ✅
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that  ✅
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit) ❌

**Comment by Clockknight** [comment]
@DaisyPalencia @alowe4 
Thank you for the feedback! I'll look into adding responsiveness for those missing platforms.

**Comment by Malcolm-Shepherd** [comment]
Last 10 daily word now shows Avg Seconds and if the user played that day.

File: wordle-web/src/App.vue [hunk 1]
```diff
@@ -1,86 +1,13 @@
...
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">Class</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
+      <NavBar class="ma-10"></NavBar>
     </div>
   </header>
 
   <RouterView />
```
Comment (mmwoodfo) [inline wordle-web/src/App.vue:8]: I don't see a site-wide theme being applied?

File: wordle-web/src/views/LeaderboardView.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { useTheme } from 'vuetify/lib/framework.mjs'
+import { LeaderboardService } from '../scripts/leaderboardService'
+import { onMounted, ref } from 'vue'
+const theme = useTheme()
+const name = ref(localStorage.getItem('name'))
+const result = ref(['', ''])
+const daily = ref(['', ''])
+onMounted(async () => {
+  result.value = await LeaderboardService.getLeaderboardFromApi()
+  daily.value = await LeaderboardService.getDailyLeaderboard()
```
Comment (mmwoodfo) [inline wordle-web/src/views/LeaderboardView.vue:58]: I think you guys are missing a day here
image

**Review by mmwoodfo — COMMENTED** [review]
You have some issues with screen resizing
image

**Review by mmwoodfo — COMMENTED** [review]
You guys seem to have lots of changes unrelated to the assignment, make sure to only include changes that are part of Assignment 4

**Pull request closed without merging**